lib/assessment.js

```diff
@@ -32,6 +38,16 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
+  var result = true;
+var newArray = [];
+  array.forEach(function(currentValue, index, array) {
+
+    if(result = callback(currentValue, index, array)){
+      newArray.push(currentValue);
+    }
+
+  });
+  return newArray;
```

Comment:
Great, you return the constructed array. This function works correctly. Full credit.